docs/conf.py

```diff
@@ -300,7 +300,8 @@ def source_replace(app, docname, source):
 
 # Dict of replacements.
 source_replacements = {
-    "{PLONE_BACKEND_VERSION}": "6.0.0",
+    "{PLONE_BACKEND_MINOR_VERSION}": "6.0",
```

Comment:
Why do we need two different versions?

```suggestion
```

Reply:
Buildout wants to know the PATCH version. 
Docker can handle MINOR version.

See `Semantic Versioning`https://semver.org/lang/de/